instrumentation/opentelemetry-instrumentation-pika/tests/test_utils.py

```diff
@@ -273,3 +279,28 @@ def test_decorate_basic_publish_no_properties(
         get_span.return_value.is_recording.assert_called_once()
         inject.assert_called_once_with(basic_properties.return_value.headers)
         self.assertEqual(retval, callback.return_value)
+
+    @mock.patch("opentelemetry.instrumentation.pika.utils._get_span")
+    def test_decorate_basic_publish_published_message_to_queue(
+        self,
+        get_span: mock.MagicMock,
+    ) -> None:
+        callback = mock.MagicMock()
+        tracer = mock.MagicMock()
+        channel = mock.MagicMock(spec=Channel)
+        exchange_name = ""
+        routing_key = "test-routing-key"
+        properties = mock.MagicMock()
+        mock_body = b"mock_body"
+        decorated_basic_publish = utils._decorate_basic_publish(callback, channel, tracer)
+        decorated_basic_publish(exchange_name, routing_key, mock_body, properties)
+
+        get_span.assert_called_once_with(
+            tracer,
+            channel,
+            properties,
+            destination=routing_key,
+            span_kind=SpanKind.PRODUCER,
+            task_name="(temporary)",
+            operation=None,
+        )
```

Comment:
Add an empty line at the end of the file

Reply:
Done